src/pages/en/guides/cms/storyblok.md

```diff
@@ -250,7 +253,7 @@ To add content, go to the content section by clicking the **Content** tab. Using
     content: Hi there! This blog post was build with Astro.
     ```
 
-Now that we have our content ready, we can switch to our Astro project and start building our blog.
+Now that you have your content ready, you can switch to Astro and start building your blog.
```

Comment:
```suggestion
Now that you have your content ready, return to your Astro project and start building your blog.
```
Is this what you mean? Leave Storyblok and go work on your Astro project?

Reply:
yes, thank you!